README.md

```diff
@@ -15,7 +15,7 @@
 *I will not accept any game automation features (macroing, etc.)*
 
 ## Features
-- Game resolution changing (about 80% complete)
+- Game resolution changing
```

Comment:
Why was this removed? The 80% complete was talking about various interfaces not centered/resized to fit resizable, but I guess it's not important